Makefile.custom.mk

```diff
@@ -3,6 +3,7 @@ generate: ## Generate kustomize patches and all helm charts
 	./hack/generate-kustomize-patches.sh
 	$(MAKE) delete-generated-helm-charts
 	kustomize build config/helm -o helm/cluster-api/templates
+	rm -v helm/cluster-api/templates/v1_configmap_watchfilter-patch.yaml
```

Comment:
According to the checks `helm/watchfilter-patch-configmap.yaml` does not exist. Is that right?

Reply:
thanks for the hint - fixed